Wikipedia/Code/AppearanceSettingsViewController.swift

```diff
@@ -53,6 +59,7 @@ open class AppearanceSettingsViewController: UIViewController, UITableViewDataSo
         tableView.contentInset = UIEdgeInsets(top: 10, left: 0, bottom: 0, right: 0);
         tableView.register(WMFSettingsTableViewCell.wmf_classNib(), forCellReuseIdentifier: WMFSettingsTableViewCell.identifier())
         tableView.register(UITableViewCell.self, forCellReuseIdentifier: AppearanceSettingsViewController.customViewCellReuseIdentifier)
+        tableView.register(UITableViewCell.self, forCellReuseIdentifier: AppearanceSettingsViewController.spacerViewCellReuseIdentifier)
```

Comment:
This & `spacerViewCellReuseIdentifier` can be removed since it's the same cell class